test/torch/tensors/test_native.py

```diff
@@ -205,3 +205,24 @@ def forward(self, x):
 
     ## Forward on the remote model
     pred = model_net(tensor_remote)
+
+
+def test_encrypt():
+    hook = syft.TorchHook(torch)
+
+    bob = syft.VirtualWorker(hook, id="bob")
+    alice = syft.VirtualWorker(hook, id="alice")
+    james = syft.VirtualWorker(hook, id="james")
+
+    x = torch.randint(10, (1, 5), dtype=torch.float32)
+    x_encrypted = x.encrypt(workers=[bob, alice], crypto_provider=james, args_fix_prec={"base": 10})
+    assert torch.all(torch.eq(x_encrypted.get().float_prec(), x))
```

Comment:
Oh we've forgotten to add a generic decrypt method!!
Would you be willing to do also this one? :) 
for MPC .decrypt() would call `.get().float_prec()` 
and for paillier, it would'nt change

Reply:
Sure. Do I submit it in another PR?
I've incorporated the suggestions mentioned.

Reply:
In this PR would be more coherent if you can do it !